submissions/EmmanuelIdiaghe/database/transportDB.sql

```diff
@@ -49,4 +49,31 @@ create table accident_status(
     foreign key(passenger_id) references passengers(id)
 );
 
-select * from accident_status;
\ No newline at end of file
+select * from accident_status;
+
+/*********************************************
+** Data Queries
+*********************************************/
+# 1. Total number of passengers who survived
+select count(*) from accident_status where accident_status > 0;
+############# Answer: 342
+
+# 2. Total number of passengers who did not survive
+select count(*) from accident_status where accident_status = 0;
+############# Answer: 549
+
+# 3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+select passengers.name_of_passenger, passengers.gender from passengers 
+inner join trip_details on passengers.id = trip_details.passenger_id 
+where age < '27' and embarkation_point like "Q%" or embarkation_point like "C%";
+
+# 4. Number of passengers who embarked at Southampton and survived
+select count(*) from accident_status 
+inner join trip_details on accident_status.id = trip_details.passenger_id 
+where accident_status > 0 and embarkation_point like "S%";
+############# Answer: 218
+
+# 5. ID, Name and Total Number of passengers who paid a fare greater than $100 and above the age of 35 had siblings or spouses on board
+select passengers.id, passengers.name_of_passenger, COUNT(*) OVER () AS total_count from passengers 
```

Comment:
Are you sure this `OVER ()` didn't throw an error. and what does it do?

Reply:
There wasn't any error. The OVER () kinda acts like the Order By for the result of the 'count(*)' and names it AS 'total_count' or whatever.

Reply:
I don't really see the reason for using `over()`. But when I ran your query, I keep getting an error though. You might want to check that out

Reply:
Again, I think these discrepancies have to do with version variations. I've attached a screenshot of the successful execution of that particular query - no errors!

![workbench](https://user-images.githubusercontent.com/29339127/99768601-57097300-2b05-11eb-8392-1c8b4df92ec4.png)

Reply:
Removing the OVER() flags an error for me! See attached

![workbench2](https://user-images.githubusercontent.com/29339127/99769655-ee22fa80-2b06-11eb-8f59-2ba45f3fda48.png)

Reply:
Hmmmmm, something is wrong. But I can't tell why. I will look into it. Kindly copy that error message and send it to me on slack